routing/localchans/manager.go

```diff
@@ -103,7 +129,45 @@ func (r *Manager) UpdatePolicy(newSchema routing.ChannelPolicy,
 		return nil
 	})
 	if err != nil {
-		return err
+		return nil, err
+	}
+
+	// Construct a list of failed policy updates.
+	for chanPoint := range unprocessedChans {
+		channel, err := r.FetchChannel(chanPoint)
+		switch {
+		case errors.Is(err, channeldb.ErrChannelNotFound):
+			failedUpdates = append(failedUpdates,
+				&lnrpc.FailedUpdate{
+					ChannelPoint: chanPoint.String(),
+					Reason:       lnrpc.FailedUpdate_NOTFOUND,
+					UpdateError:  "not found",
+				})
+
+		case err != nil:
+			failedUpdates = append(failedUpdates,
+				&lnrpc.FailedUpdate{
+					ChannelPoint: chanPoint.String(),
+					Reason:       lnrpc.FailedUpdate_INVALID,
```

Comment:
What do we mean by `INVALID` here? Would `INTERNAL_ERR` be a better name? Since we're failed for an unexpected reason?

Reply:
I think INTERNAL_ERR is great.